.github/pull_request_template.md

```diff
@@ -1,49 +1,69 @@
 ## Motivation
-
 <!--
 Thank you for your Pull Request.
-How does this change improve Zebra?
+Does it close any issues?
 -->
+_What are the most important goals of the ticket or PR?_
+
+
+### PR Author Checklist
+
+Check before marking the PR as ready for review:
+  - [ ] Will the PR name make sense to users?
+    - [ ] For significant changes: Is there a summary in the CHANGELOG?
+    - [ ] Does the PR have a priority label?
+  - [ ] Have you added or updated tests?
+  - [ ] Is the documentation up to date?
```

Comment:
```suggestion
#### Check before marking the PR as ready for review:
  - [ ] Will the PR name make sense to users?
  - [ ] Does the PR have a priority label?
  - [ ] Have you added or updated tests?
  - [ ] Is the documentation up to date?

##### For significant changes:
  - [ ] Is there a summary in the CHANGELOG?
  - [ ] Do all of the changes in this PR need to be merged in together?
        (Are there no change that should be split out into a separate PR?)
```

Reply:
Sure, let's ask the question directly?

Optional: Could we include a reminder to try and split apart large PRs?

```suggestion
#### Check before marking the PR as ready for review:
  - [ ] Will the PR name make sense to users?
  - [ ] Does the PR have a priority label?
  - [ ] Have you added or updated tests?
  - [ ] Is the documentation up to date?

##### For significant changes:
  - [ ] Is there a summary in the CHANGELOG?
  - [ ] Can these changes be split into multiple PRs?
```